spec/__snapshots__/aboutApp.spec.ts.snap

```diff
@@ -189,7 +189,9 @@ ShallowWrapper {
           "getChildContext": Object {
             "calledByRenderer": false,
           },
-          "getDerivedStateFromProps": true,
+          "getDerivedStateFromProps": Object {
+            "hasShouldComponentUpdateBug": false,
```

Comment:
what does this property mean?

Reply:
Based on your question I change the snapshot. It really doesn't make sense to have this in the snapshot. Take a look: https://github.com/symphonyoss/SymphonyElectron/pull/596